app/scripts/views/components/barChart/barChartTemplate.js

```diff
@@ -201,9 +202,16 @@
       if (((this.data.max - this.data.min) > 1000) && (this.data.min > 1)) {
         this.settings.showLogScale = true;
       }
-      this.initChart(this.settings.showLogScale);
-      this.updateShowSurvivalIcon();
-      this.$dispatch('data-loaded', this.attributes.group_id, this.chartDivId);
+      
+      if (isNaN(this.data.min) || isNaN(this.data.max)) {
```

Comment:
Actually, it won't. NaN can be calculated so that's why when `findExtremes ` returns [NaN, NaN, NaN] the line 202 doesn't throw an exception. But you do have a good point, I should check NaN before line 202